iso-browser.js

```diff
@@ -6,7 +6,7 @@ const isoCrypto = typeof crypto === 'undefined' ? null : crypto
 /**
  * @type {function(number):ArrayBuffer}
  */
-const cryptoRandomBuffer = isoCrypto !== null
+const cryptoRandomBuffer = (isoCrypto !== null && typeof isoCrypto.getRandomValues === 'function')
```

Comment:
Hi @dtilik ,

This change makes sense. Thanks for that! But can you please revert the other changes (incrementing the version and changing package-lock)? This is done by my publishing script.